test/lit/passes/dae-gc.wast

```diff
@@ -211,15 +211,15 @@
  ;; NOMNL-NEXT:  )
  ;; NOMNL-NEXT: )
  (func $call-bar
-  ;; Call with nulls. Mixing nulls is fine as they all have the same value, and
-  ;; we can optimize (to the LUB of the nulls). However, mixing a null with a
-  ;; reference stops us in the second param.
+  ;; Call with nulls. Mixing nulls is fine as they all have the same type and
+  ;; value. However, mixing a null with a reference stops us in the second
+  ;; param.
   (call $bar
-   (ref.null i31)
-   (ref.null data)
+   (ref.null none)
+   (ref.null none)
```

Comment:
Is there no value in keeping tests for us treating all nulls the same? I guess we are assuming that we turn them into the same `none` during parsing - how is that tested?

Reply:
I guess we test the parsing in every test that uses a non-bottom type with a null. But in any particular test, it's better to use `none` so that the difference between input and output is limited to things that the test is trying to show. If we ever get to a point where most tests are using bottom types in their input, then we should add a dedicated test for the other cases.

Reply:
Sounds good.